server/message_hooks.go

```diff
@@ -542,7 +542,7 @@ func (p *Plugin) handlePromptForConnection(userID, channelID string) {
 	}
 
 	if time.Until(timestamp) < -time.Hour*time.Duration(promptInterval) {
-		p.sendBotEphemeralPost(userID, channelID, "Your Mattermost account is not connected to MS Teams so your activity will not be relayed to users on MS Teams. You can connect your account using the `/msteams-sync connect` slash command.")
+		p.sendBotEphemeralPost(userID, channelID, "Your Mattermost account is not connected to MS Teams so your activity will not be relayed to users on MS Teams. You can connect your account using the `/msteams connect` slash command.")
```

Comment:
Are we standardizing on "MS Teams" versus "Microsoft Teams" or "MS-Teams" ?

Reply:
Good question! I'm hoping @calebroseland and @asaadmahmood can help us with all the bot interactions as well as word smithing going forward.